src/lib/loaders/loaders_with_authentication/causality.ts

```diff
@@ -33,20 +25,23 @@ export const causalityLoaders = (_accessToken, userID) => {
       method: "POST",
       body,
       headers: {
-        Authorization: `Bearer ${token}`,
+        Authorization: `Bearer ${CAUSALITY_TOKEN}`,
```

Comment:
@dzucconi - this should fix all of that weirdness you were seeing when trying to repo the bug. Lots should now all return (assuming you've updated your ENV with the value in 1pass)